brain/compartmentReportView.cpp

```diff
@@ -95,45 +95,18 @@ std::future<CompartmentReportFrame> CompartmentReportView::load(
     return _impl->readerImpl->threadPool.post(loadFrameTask);
```

Comment:
This can be done in a separate pull request. This one was just to make the API asynchronous.